### ISSUE TITLE
Can't fetch vendor: Bower error

### ISSUE THREAD

**Issue by Geolim4** [avanzu/AdminThemeBundle#127]
Hi
When i try to fetch the vendor, It output this error:

`
php bin/console avanzu:admin:fetch-vendor -e=prod
[Executing] bower  install
[Error] bower                     invalid-meta The "name" is recommended to be lowercase, can contain digits, dots, dashes
`

**Comment by Geolim4** [comment]
Also i can't build the asset, the command try to call uglify like Unix env:

`/usr/bin/env uglifyjs -o`

As i'm on windows the **bower_bin** option is working well for me, why not making the same for **uglifyjs** and **uglifycss** ?

The full command output (i replaced sensitive data by **0**):

``` bash
λ php bin/console avanzu:admin:build-assets
/usr/bin/env uglifyjs -o F:\UwAmp\www\000000-bundle\000000Bundle\vendor\avanzu\admin-theme-bundle\Resources/public/static/dev/scripts/modernizr.js F:\UwAmp\www\000000-bundle\000000Bundle\vendor\avanzu\admin-theme-bundle/Resources/public/vendor/modernizr/modernizr.js

Le chemin d'accès spécifié est introuvable.
```

**Comment by Geolim4** [comment]
Ok the binaries can be specified as a cli option. I have to made some research to find this.
Anyway the style command does not output anything, i made a Pr for that: #128

**Comment by Geolim4** [comment]
Closed as merged in #128
